typescript/package.json

```diff
@@ -23,8 +23,8 @@
         "storybook": "storybook dev -p 6006",
         "build-storybook": "storybook build",
         "storybook:test": "test-storybook",
-        "docker:build": "docker build --tag wsc_story_test - < Dockerfile.test",
-        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=.,target=/test --network host --ipc=host wsc_story_test",
+        "docker:build": "docker build --tag wsc_story_test --file Dockerfile.test .",
+        "docker:storybook:test": "docker run -it --rm --mount type=bind,source=$(pwd)/packages,target=/test/packages --network host --ipc=host wsc_story_test",
```

Comment:
$(pwd) does not work on windows with git bash shell :/

Reply:
What error do you get?

Does it work if you replace `$(pwd)` with `%cd%`?

Reply:
Yes, it works with %cd%.
Thus we can either solve the issue by using `cross-env` package, or by using linux vs. windows specific scripts...